common/persistence/cassandra/cassandraMetadataPersistenceV2.go

```diff
@@ -92,14 +92,24 @@ const (
 )
 
 type (
-	cassandraMetadataPersistenceV2 struct {
+	// CassandraMetadataPersistenceV2 domain metadata v2 impl
+	// TODO make this struct private, once domain migration admin command is finished
```

Comment:
Another way to consider is to keep this private, but create a public function which takes interface MetadataManager, and cast to cassandraMetadataPersistenceV2 impl inside, then use this function in cli to do migrate

Reply:
+1 like we discussed offline